BridgeCareApp/BridgeCare/DataAccessLayer/AttributeNames.cs

```diff
@@ -0,0 +1,39 @@
+﻿using BridgeCare.ApplicationLog;
+using BridgeCare.Interfaces;
+using BridgeCare.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+
+namespace BridgeCare.DataAccessLayer
+{
+    public class AttributeNames : IAttributeNames
+    {
+        public AttributeNames() { }
+
+        public IQueryable<AttributeNameModel> GetAttributeNames(BridgeCareContext db)
+        {
+            IQueryable<AttributeNameModel> rawQueryForData = null;
+
+            var select = ("SELECT ATTRIBUTE_ as Name FROM Attributes_");
+
+            try
+            {
+                rawQueryForData = db.Database.SqlQuery<AttributeNameModel>(select).AsQueryable();
+
```

Comment:
Use CodeMaid to eliminate extraneous spacing and to properly order fields and methods, etc.